cloudbuild-production.yaml

```diff
@@ -11,6 +11,7 @@ steps:
   - -c
   - >
     docker build
+    --build_arg COMMIT_SHA=$COMMIT_SHA
```

Comment:
This has to be done on the infra repo! 

https://github.com/learningequality/infrastructure/pull/661

done there, applied to `unstable` only so far. No harm in adding it to this file, but this file is useless.